src/backend/distributed/commands/utility_hook.c

```diff
@@ -370,6 +370,18 @@ multi_ProcessUtility(PlannedStmt *pstmt,
 
 				ConvertNewTableIfNecessary(createStmt);
 			}
+
+			if (context == PROCESS_UTILITY_TOPLEVEL &&
+				IsA(parsetree, AlterObjectSchemaStmt))
```

Comment:
kind of wondering about the meaning of only doing this when UtilityHookLevel == 1.

I guess it's ok, but seems like a strange condition.